### ISSUE TITLE
Ember tests error

### ISSUE THREAD

**Issue by j-carvalho** [ember-cli/ember-cli#5084]
version: 1.13.11
node: 4.1.1
npm: 2.14.10
os: darwin x64

Also occurs on Windows.

Build is successful but when the tests are running I get this message:

Uncaught Error: Assertion Failed: The tests file was not loaded. Make sure your tests index.html includes "assets/tests.js".

**Comment by nightire** [comment]
compare your `tests/index.html` with this version: https://github.com/ember-cli/ember-cli/blob/master/blueprints%2Fapp%2Ffiles%2Ftests%2Findex.html

you might missing some changes when upgrading your app

**Comment by j-carvalho** [comment]
But we didn't upgrade our app.

**Comment by j-carvalho** [comment]
Althought just placing this line solved the problem:
 <script src="assets/tests.js"></script>

Thank you @nightire

**Comment by stefanpenner** [comment]
> But we didn't upgrade our app.

if you are using 1.13.11 you must have updated your app, because 1.13.11 came out yesterday.

**Comment by dejongnj** [comment]
This might be a little late to the game, but I had the same problem and came here looking for a solution. None of the above worked, but I figured out a solution that hopefully will help someone else:

I already had an app/server running so I used ember init to change my app to an ember app. This seems to assume that your module names will be the same as the folder you're creating the app in and then in your index.html use that name for e.g. src="{{rootURL}}assets/root-folder-name.js".

If you change the modulePrefix (which defaulted to "root-folder-name") to what you actually want, e.g. "my-custom-app", then you'll need to make those changes in the index.html in both the app and the test folders. e.g. src="{{rootURL}}assets/my-custom-app.js".

This seemed to work for me.

**Comment by kmccullough** [comment]
Recently I came up against this error while upgrading. The error was inexplicably output on my test element, polluting my output, despite the tests passing so long as they ignored this extra text. I finally found the issue in `environment.js` where I needed to add the following line:

`ENV.APP.rootElement = '#ember-testing';`

This was a cryptic error for this particular mistake. I hope this helps someone.
